src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_button.scss

```diff
@@ -28,97 +37,107 @@ It may have the following states:
 -   ng-disabled or :disabled
 
 ```html_example
-<a class="button">Link</a>
-<button type="button">Button</button>
-<input type="submit" value="Submit" />
-<input type="submit" value="Disabled" disabled="disabled" />
+<a class="button" href="">Link</a>
+<button type="button" class="button">Button</button>
+<input type="submit" class="button" value="Submit" />
+<input type="submit" class="button" value="Disabled" disabled="disabled" />
 ```
 */
 
 /*doc
 ---
-title: call-to-action modifier
-name: button.m-call-to-action
+title: button-cta
```

Comment:
the title should be descriptive rather than technical. What about "Call-to-action button"